BarricadeKit/Core/ResponseSet/ResponseSetEvaluation.swift

```diff
@@ -32,6 +32,12 @@ extension ResponseSetEvaluation {
         }
     }
 
+    public static func contains(_ string: String) -> ResponseSetEvaluation {
```

Comment:
This is great 👍 One request though... if you take a look at `ResponseSetEvaluationTests.swift` there should be a couple of test cases for testing success and failure of the existing `suffix` evaluation. Would you add similar test cases for this new evaluation?

Reply:
Added new test case for the new evaluations in the updated PR.